portfolio/src/main/webapp/script.js

```diff
@@ -52,41 +52,56 @@ function addRandomDestination() {
 }
 
 
+async function checkLoggedIn() {
+  const loginResponse = await fetch('/login', {method: 'POST'});
+  const loginJson = await loginResponse.json();
+  addLogInOutButton(loginJson.url, loginJson.isLoggedIn);
+  console.log(loginJson.isLoggedIn);
```

Comment:
Printing a boolean isn't that nice. Can you either add some context or remove this?